asmcli/commands/validate.sh

```diff
@@ -64,6 +68,12 @@ validate_dependencies() {
 
   if can_register_cluster; then
     register_cluster
+    exit_if_cluster_unregistered
+    exit_if_hub_membership_is_empty
```

Comment:
nit: not sure if this is required anymore. From ASM 1.13, fleet registration is mandatory

Reply:
These two statements check potential errors (e.g., register fails, hub membership is empty). Though registration is mandatory, the registration may fail. If these two statements detect errors, they will exit.